lib/faraday/log_filter/filter.rb

```diff
@@ -0,0 +1,40 @@
+module Faraday
+  module LogFilter
+    class Filter
+      def initialize(params_to_filter)
+        @params_to_filter   = Array(params_to_filter)
+        @params_to_truncate = @params_to_filter.last.is_a?(Hash) ? @params_to_filter.pop : {}
+      end
+
+      def filter_url(url)
+        filtered_query = filter(CGI::parse(url.query.to_s))
+        url.query      = filtered_query.map { |params| params.join("=") } * "&"
```

Comment:
it seems like using the `*` operator after an array of strings does the same thing as `join`, right? What do you think about using `join` here for consistency?